### ISSUE TITLE
[feature] Configurable via UI

### ISSUE THREAD

**Issue by LazyTarget** [redstone99/hass-alert2#6]
Would be nice if this could be configurable via the UI and it's Alert-entity to be attached to a device.

**Comment by redstone99** [comment]
Hi, thanks for the suggestion. I'm looking into making alerts support UI configuration.  I may start off with a simple text input box that lets you type YAML.  I'm guessing you'd prefer a UI experience that's not YAML and is more structured?

Also, what do you mean by attaching an alert entity to a device?  Can you give me an example?
-J

**Comment by LazyTarget** [comment]
Yeah a structured UI option would be optimal. But a yaml editor works as well, that would eliminate the need to Reload/Restart HA

To clarify, I mean in the way that some helpers work. For example the Template sensor helper that can be created and managed through the UI. It has a field for Device, which makes the created helper entity accessible from that device page.

**Comment by LazyTarget** [comment]
> Thanks, I'll look into device connections, and was about to look into UI editing coincidentally.
Wonderful!

> Also, the recent release of Alert2 now supports YAML reloading on the fly (via "Developer tools" -> "YAML" and clicking on "Alert2" to reload)
Oh, I missed that, thats great!

**Comment by redstone99** [comment]
I recently released v1.8, which adds (text-based) support for creating & editing alerts via the UI.  Does that meet your needs for this feature request?

**Comment by redstone99** [comment]
@LazyTarget , I recently released UI alert editing support in v1.8.  Does that meet what you were looking for?  If so, I can close this item.
J

**Comment by redstone99** [comment]
Closing this, but let me know if there's more to do - happy to reopen.
J
